contracts/mechs/nevermined/BalanceTrackerNvmSubscriptionNative.sol

```diff
@@ -38,13 +32,13 @@ error Overflow(uint256 provided, uint256 max);
 /// @param amount Value amount.
 error NoDepositAllowed(uint256 amount);
 
-contract BalanceTrackerNvmSubscription is BalanceTrackerBase {
-    event SubscriptionSet(address indexed token,  uint256 indexed tokenId);
-    event WithdrawSubscription(address indexed account, address indexed token, uint256 indexed tokenId, uint256 amount);
+/// @title BalanceTrackerFixedPriceNative - smart contract for tracking mech and requester subscription balances based on native token
+contract BalanceTrackerNvmSubscriptionNative is BalanceTrackerFixedPriceNative {
+    event SubscriptionSet(address indexed token, uint256 indexed tokenId);
     event RequesterCreditsRedeemed(address indexed account, uint256 amount);
 
-    // TODO: setup, taken from subscription?
-    uint256 public constant NVM_FEE = 100;
+    // Credit to token ratio
+    uint256 public immutable creditTokenRatio;
```

Comment:
This has to be in place - there's no on-chain way to see how much wei goes for one credit, this is all NVM internal info input during the subscription creation